.github/workflows/.lint.yaml

```
@@ -5,56 +5,345 @@ permissions:
-        uses: actions/checkout@93cb6efe18208431cddfb8368fd83d5badbf9bfd # v5
+        uses: actions/checkout@v4
         with:
-          token: ${{ secrets.GH_TOKEN }}
           fetch-depth: 0
+      - name: Install trufflehog
+        uses: jaxxstorm/action-install-gh-release@v1.10.0
+        with:
+          repo: trufflesecurity/trufflehog
+          cache: enable
+      - name: Git Secret Scanning
+        run: >
```

actionlint (1.7.9)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)